src/heitech.zer0mqXt.core/Infrastructure/XtResult.cs

```diff
@@ -48,7 +48,7 @@ internal static XtResult<T> Success(T result, string operation = "request")
 
         public override string ToString()
         {
-            var success = IsSuccess ? " succeeded " : " failed";
+            var success = IsSuccess ? "succeeded " : "failed";
```

Comment:
```suggestion
            var success = IsSuccess ? "succeeded" : "failed";
```

There is also a trailing space.

Reply:
done